syntax = "proto3";

package server_service;

// Interface exported by the server.
service ServerServiceInterface 
{
	rpc SendMessage(MessageRequest) returns (MessageResponse) {}
}

// Message received from the client to the server
message MessageRequest
{
	string username = 1;
	string message = 2;
}

// Message sent from the server to the client
message MessageResponse
{
	string username = 1;
	string response = 2;
}